app/assets/javascripts/components/articles/PaginatedArticleControls.jsx

```diff
@@ -19,7 +19,7 @@ export const PaginatedArticleControls = ({ showMore, limitReached }) => {
   const filteredArticles = useSelector(getArticlesByTrackedStatus);
   const currentPage = useSelector(state => state.articles.currentPage);
   const pageRange = getPageRange(currentPage, filteredArticles.length);
-  const totalEditedArticles = getPageRange(currentPage, filteredArticles.length).split('-')[1];
+  const totalFilteredArticles = Math.min(filteredArticles.length, currentPage * ARTICLES_PER_PAGE);
```

Comment:
Why is this not just filteredArticles.length? In what circumstance would currentPage * ARTICLES_PER_PAGE be smaller than the length of `filteredArticles`?

Reply:
In case if there are more than 100 ` filteredArticles` for a particular filter and since only 100 articles can be shown per page only.
Also, i think i made a mistake here because then only  `ARTICLES_PER_PAGE` will be enough instead of  `currentPage * ARTICLES_PER_PAGE`. 
Then it will simply be  - `Math.min(filteredArticles.length,  ARTICLES_PER_PAGE)`.